client/templates/admin/postEditor/postEditor.html

```diff
@@ -37,8 +37,7 @@
             <textarea
               id="content-input"
               class="article-description"
-              name="content"
-              placeholder="Enter post body content">{{ model.content }}</textarea>
+              name="content">{{ model.content }}</textarea>
```

Comment:
I agree!

> 19. mai 2016 kl. 15.45 skrev Sebastian Sastre <notifications@github.com>:
> 
> In client/templates/admin/postEditor/postEditor.html <https://github.com/dagvonkr/hof3/pull/12#discussion_r63880405>:
> 
> > @@ -37,8 +37,7 @@
> >              <textarea
> >                id="content-input"
> >                class="article-description"
> > -              name="content"
> > -              placeholder="Enter post body content">{{ model.content }}</textarea>
> > +              name="content">{{ model.content }}</textarea>
> I think this way of adding the content won't work. It will add the html as it where text and we do want the DOM to insert the html elements stored in the content. I'm adding a commit to make this work @dagvonkr <https://github.com/dagvonkr>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/dagvonkr/hof3/pull/12/files/5104f612750e6a9b0ac08cfc2ac224cd39015426#r63880405>